test/cppsuite/test_harness/workload/random_generator.h

```diff
@@ -60,11 +64,16 @@ class random_generator {
 
     private:
     random_generator();
+    std::uniform_int_distribution<> &get_distribution(characters_type type);
+    const std::string &get_characters(characters_type type);
 
     std::mt19937 _generator;
-    std::uniform_int_distribution<> _distribution;
-    const std::string _characters =
+    std::uniform_int_distribution<> _alphanum_distrib, _alpha_distrib;
+    /* Alphanumeric characters without the special ones. */
+    const std::string _pseudo_alphanum =
       "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
+    /* Alphabet characters. */
+    const std::string _alphabet = "abcdefghijklmnopqrstuvwxyz";
```

Comment:
Added alphabet characters only so we can either generate random strings either from alphabet characters only or pseudo alphanum ones.